src/index.js

```diff
@@ -5,19 +5,32 @@ function startsWith (key, val) {
   return key.indexOf(val) === 0;
 }
 
+function isAttribute (key) {
+  return key === 'attributes';
+}
+
 function shouldBeAttr (key, val) {
-  return startsWith(key, 'aria-') || startsWith(key, 'data-');
+  return startsWith(key, 'aria-') || startsWith(key, 'data-') || isAttribute(key);
 }
 
 function handleFunction (Fn) {
   return Fn.prototype instanceof HTMLElement ? new Fn() : Fn();
 }
 
+function setAttr (node, attrName, attrValue) {
```

Comment:
> I think it might be good to just make this a breaking change 

? why BC? it's clearly a feature. Currently there is no option to set attributes via bore within test, only via strings ( which doesnt work either #12 )

Overall as I re-thought the approach, overriding `attributes` feels just wrong. I would rather go as you've suggested with `attrs` ( also because from Typescript perspective, native HTLMElement.attributes cannot be overriden - with `attrs` smooth typechecking can be provided )

> Maybe at the same time we can do events?

I agree, but that should be addressed in next PR